qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: coderlite
+domain: ceo
+seed_examples:
+- answer: | Lisa Su is related to Nvidia CEO Jen Hsun Huang
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!